api/src/main/java/org/bahmni/module/bahmni/ie/apps/service/impl/BahmniFormTranslationServiceImpl.java

```diff
@@ -236,7 +236,7 @@ private JSONObject getTranslationJsonFromFile(String formName, String formVersio
 	private JSONObject getTranslations(File translationFile) {
 		String fileContent;
 		try {
-			fileContent = translationFile.exists() ? FileUtils.readFileToString(translationFile) : "";
+			fileContent = translationFile.exists() ? FileUtils.readFileToString(translationFile, "UTF-8") : "";
```

Comment:
Can we write any supporting tests for this?

Reply:
Done. Please check.